pkg/registry/scheduling/priorityclass/storage/storage.go

```diff
@@ -56,3 +59,14 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"pc"}
 }
+
+// Delete ensures that system priority classes are not deleted.
+func (r *REST) Delete(ctx genericapirequest.Context, name string, options *metav1.DeleteOptions) (runtime.Object, bool, error) {
+	for _, spc := range scheduling.SystemPriorityClasses() {
+		if name == spc.Name {
+			return nil, false, fmt.Errorf("%v is a system prioirty class and cannot be deleted", name)
```

Comment:
priority

Reply:
does this need to be an API error (e.g. NewForbidden or NewBadRequest)?

Reply:
Right. Good catch. Fixed it.